tools.go

```diff
@@ -0,0 +1,21 @@
+// +build tools
+
+package agones
+
+/*
```

Comment:
We shouldn't leave this lying around :smile:

Reply:
The intention for documentation. That way when we add or remove tools, we can easily know where they are used.